Source/UI/NewUI/CommonUI/View/Component/InputView.swift

```diff
@@ -208,7 +208,7 @@ extension InputView: TextFieldViewDelegate {
     func textFieldShouldEndEditing(textField: UITextField, replacementString: String) -> Bool {
         let shouldEndEditing = delegate?.textFieldShouldEndEditing(textField: textField, replacementString: replacementString) ?? true
         if shouldEndEditing {
-            textFieldContainer.layer.borderColor = style?.textfield.normalBorderColor.cgColor
+            textFieldContainer.layer.borderColor = (style?.error?.isHidden ?? true) ? style?.textfield.normalBorderColor.cgColor : style?.textfield.errorBorderColor.cgColor
```

Comment:
Does this need any updates to unit tests ?

Reply:
On the face of it, being just UI related (which colour is shown for certain config), would be tempted to say this may be covered by snapshots. As we don't have those, not sure if I'd worry too much of unit testing this, there's no easy ways to do it.

But if this is being done already, sure extending tests to cover for this would be nice.